plugins/sqlfluff-templater-dbt/test/fixtures/dbt/dbt_project/models/my_new_project/incremental.sql

```diff
@@ -12,4 +12,4 @@ where not products._fivetran_deleted
     {% if true -%}
     and products.valid_date_local >= (
         select max(valid_date_local) from {{ this }})
-    {% endif -%}
+    {% endif %}
```

Comment:
This started breaking after the recent L009 PR was merged. IIUC, using whitespace control at the end of the file caused all the trailing newlines to be (correctly) removed by Jinja, thus L009 would report an issue. I feel comfortable with this change, i.e. that the behavior was correct and that this fix makes sense.